### ISSUE TITLE
Please choose an open source license for System.Reflection.Abstractions

### ISSUE THREAD

**Issue by schlotter** [MarcelRoozekrans/System.Reflection.Abstractions#1]
Hello Marcel,

Many thanks for creating `System.Reflection.Abstractions`! It is really useful, hence I would like to use it in one of my projects. Could you please place it under a permissive license (like MIT, Apache, BSD)?

I have learnt from [GitHub's help](https://help.github.com/en/github/creating-cloning-and-archiving-repositories/licensing-a-repository#choosing-the-right-license) that as long as a license is not chosen others can unfortunately not use your useful project:

> You're under no obligation to choose a license. However, without a license, the default copyright laws apply, meaning that you retain all rights to your source code and no one may reproduce, distribute, or create derivative works from your work. If you're creating an open source project, we strongly encourage you to include an open source license.

Many thanks and best regards,
Christian

**Comment by MarcelRoozekrans** [comment]
@schlotter Yeah sure! Problem is that we cannot push it to nuget repo because the namespace is protected.

**Comment by schlotter** [comment]
Thank you! Interesting, I did not know about prefix reservation.

I see two ways:
1. Choose another name. This might make it more difficult for users to find the package.
2. Try the [_Package ID Prefix Reservation Dispute Policy_](https://docs.microsoft.com/nuget/nuget-org/id-prefix-reservation).

Somehow [System.Configuration.Abstractions](https://www.nuget.org/packages/System.Configuration.Abstractions/) got into NuGet.org, although it also contained a reserved prefix...

**Comment by schlotter** [comment]
I have sent an email to NuGet.org support, and there is no way to get the package under its current name to NuGet.org. Some older packages exist in reserved prefixes because they occupied the spot before ID Prefix Reservation was introduced.

So what about renaming? Maybe `Sys.Reflection.Abstractions`, or `MR.System.Reflection.Abstractions`?
